manager/controllers/utils/configuration.go

```diff
@@ -26,6 +26,7 @@ const (
 	SecretProviderURL                   string = "SECRET_PROVIDER_URL"
 	SecretProviderRole                  string = "SECRET_PROVIDER_ROLE"
 	VaultModulesRole                    string = "VAULT_MODULES_ROLE"
+	CatalogProviderName                 string = "CATALOG_PROVIDER_NAME"
```

Comment:
Do we already have such environment variable in the config?

Reply:
Yes, we have a name for all providers (catalog, policy, credentials)

Reply:
Reverted.